### PR TITLE
Problem:(cro-672) jailing test passes with compiling failure

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: if compiling fails, stop further integration test

### CODE REVIEW

**Comment by leejw51crypto** [comment]
checked with drone

**Comment by leejw51** [comment]
also make it use nix-compiled binaries

File: integration-tests/jail/docker-compose.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ services:
     volumes:
       - ./disk:/root/disk
       - ./disk/config0:/root/config
+      - /nix:/nix
```
Comment (leejw51) [inline integration-tests/jail/docker-compose.yml]: no,  running inside docker fine

File: integration-tests/jail/rust.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ];
+
+ shellHook = ''
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "zeromq-[0-9\.]+$" | head -n1)/lib":$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "clang-[0-9\.]+$" | head -n1)/lib":$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "libffi-[0-9\.]+$" | head -n1)/lib":$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "llvm_6.0" | head -n1)/lib":$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "gcc-6.5.0-lib" | head -n1)/lib":$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="/nix/store/$(ls /nix/store | grep -E "libedit" | head -n1)/lib":$LD_LIBRARY_PATH
+  '';
```
Comment (leejw51) [inline integration-tests/jail/rust.nix]: i borrowed this from calvin's integration code

**Comment by leejw51** [comment]
for compiling, it's not necessary.
i'll remove it.

**Comment by tomtau** [comment]
closing this PR, as it's been stalled for 15 days -- @leejw51crypto feel free to re-open

**Pull request closed without merging**